.github/ISSUE_TEMPLATE.md

```diff
@@ -6,6 +6,8 @@ _Before filing a new issue, please **provide the following information**._
 > - **Which operating system?**: Windows / MacOS / Linux
 > - **How installed?**: via installer / homebrew / binaries / from source
 > - **Are you fully synchronized?**: no / yes
+> - **Which network are you connected to?**: ethereum / ropsten / kovan / ...
```

Comment:
s/ethereum/foundation/ ?

Reply:
I think Ethereum is still a valid and well-understood reference to the primary public network. :)

Reply:
100%, just mentioned it since we use "foundation" as the main chain name in our software.